espnet/utils/training/batchfy.py

```diff
@@ -317,7 +321,7 @@ def make_batchset(data, batch_size=0, max_length_in=float("inf"), max_length_out
         okey = "output"
         batch_sort_key = "output"
         batch_sort_axis = 1
-        iaxis = 1
+        assert iaxis == 1
```

Comment:
Sweat!

Reply:
Can you also add a line: `assert oaxis == 1`?

Reply:
Can you confirm this? I thought iaxis is 1 and oaxis=0 according to [line1](https://github.com/espnet/espnet/blob/f7d2100979f1e4f2e8d725bb1af3c67c1ad20271/espnet/utils/training/batchfy.py#L305) and [line2](https://github.com/espnet/espnet/blob/f7d2100979f1e4f2e8d725bb1af3c67c1ad20271/espnet/utils/training/batchfy.py#L320).

Reply:
Sorry, you're right. `assert oaxis == 0` is correct.